ONSdigital/eq-questionnaire-runner
# Adds a transform to get the label for an answer value

### What is the context of this PR?
Adds a transform to get the label for an answer value

### How to review 

Single answer (placeholder)
1. Load the the survey `test_placeholder_option_label_from_value.json`
2. Select a single radio answer
3. The first option of radio is a placeholder piped from metadata
3. Check that the question text on the next page contains the label of the question you selected (If you selected `ESSENTIAL SERVICES LTD` the text should contain `ESSENTIAL SERVICES LTD. (piped)`, which is a placeholder for the label in the schema). Submit.
4. Check that the summary page contains the single checkbox answer and the recovery time answer. 


### Checklist
